### DISCUSSION TITLE
Suggestion

### DISCUSSION THREAD

**Discussion by AlABarazi** [crewAIInc/crewAI#1226] [Q&A]
I have been experimenting with crewai for some day. It is really amazing library.
However, given the high numbers of bugs and unexpected behavior that I have faced, I am hesitant to use it for projects. I suggest that you test each example with at least two models (or more) GPT-4 and llama3 (local). In my opinion, this would help the user for your examples to save debugging time and put more time on building something useful. Given the iterative process of using crewai and the lake of opinionated implementation, starting any new project with the current state of crewai may require a lot of time for debugging and trail and error. Which is  cumbersome process.

**Comment by theCyberTech** [comment]
Hi, What issues have you faced specifically for models?

**Comment by AlABarazi** [comment]
Hi,
I tested now most of the examples using crewai .036.1. I needed to add expected_output to tasks objects as it was not required in the old versions.  The quality of results rely on the type and the the number of parameters of LLM used.
I am not able to test https://github.com/crewAIInc/crewAI-examples/tree/main/landing_page_generator as i dont' have tailwindui.com/templates (they require payment). Could you help how could I run this examples ? And what is the right channel to seek help for building new projects?
